yarn-project/aztec-cli/README.md

```diff
@@ -1,44 +1,44 @@
 # Aztec CLI Documentation
 
-The Aztec CLI `azti` is a command-line interface (CLI) tool for interacting with Aztec. It provides various commands for deploying contracts, creating accounts, interacting with contracts, and retrieving blockchain data.
+The Aztec CLI `aztec-cli` is a command-line interface (CLI) tool for interacting with Aztec. It provides various commands for deploying contracts, creating accounts, interacting with contracts, and retrieving blockchain data.
```

Comment:
+1 on this being the less surprising name (which people can always alias as needed)